### DISCUSSION TITLE
Performance issues in hybrid search with Azure hosted Meilisearch 1.8.0 instance

### DISCUSSION THREAD

**Discussion by miguelisidoro** [meilisearch/product#753] [Experimental features]
Hello,

In a Azure hosted Meilisearch 1.8.0 instance, we are having some performance problems. Hybrid search is taking a long time, from 200ms up to 25s (once it took 25s and some other times it took between 10 and 20s to get a response) . Longer times happen when no requests are made for a long period of time (15 to 30m).
If we continuously make requests, the responses are usually under 500ms which is still considerable.

I am using the following query with the endpoint {{url}}/indexes/{{indexUID}}/search 

{
    "q": "benefits, company",
    "filter": "tenant = 'Default' AND (targetAudience = 'Global' OR targetAudience = 'Administrator' OR targetAudience = 'Global Admin' OR targetAudience = '4ea3gq7r13q34sr6sbey4105nw')",
    "hybrid": {},
    "vector": [
            ]
}

Any ideas on how to improve performance?

Thanks

**Comment by macraig** [comment]
Issue being discussed in the [community Discord](https://discord.com/channels/1006923006964154428/1250461912727293973/1250461912727293973)

**Reply by miguelisidoro** [reply]
Hi,

Although it is being discussed in the community, it is very important for me since the degree of importance of this issue, to have feedback from the Meilisearch team since they know the product in more detail.

Is this possible?

Thanks
